lib/elliptic/use-precomputed.js

```diff
@@ -0,0 +1,22 @@
+var curves = require('./curves');
+
+function usePrecomputed(precomputed) {
```

Comment:
Since the precomputed curves are stored inside the package, why do we make the caller specify them with an awkward internal require? Why not have `precomputed` be a string or some other constant which identifies the curve, and the package handles finding the curve's JSON file.

Reply:
it's so the caller can decide what gets bundled without resorting to dynamic requires. I agree that it's awkward, and i'm open to a simpler way you know one :)

Reply:
Ah thank you, that makes more sense. 

What if the actual precomputed curve files could be packaged separately? This package would have the logic of how to use the precomputed curves, while a separate NPM package holds the curve JSON files themselves, using `elliptic` as a devDependency to generate them.

I don't think we need to do that right now though, just a future todo.

Reply:
yes, we could put them in a separate package, say `@exodus/precomputed-curves`, but the tests would be a bit harder to do than directly inside `elliptic`

we could do a mix of the two: gen precomputed curves and test them here, but re-export a cleaner api for `usePrecomputed` from that separate package